content/en/contribute/code/android/releasing.md

```diff
@@ -30,16 +32,5 @@ All Medic's android projects automatically build, sign, and release builds via G
 
 1. Repeat step 2-3 from the above [alpha release section](#alpha-for-release-testing) with the naming convention `v<major>.<minor>.<patch>`.
 2. The CI build for the tag will create a new draft release on GitHub.  Include the release notes from the CHANGELOG in the description of the release and publish the release on GitHub.
-3. [Publish]({{< ref "core/guides/android/publishing" >}}) in the Play Store, F-Droid or whatever channel is used for publishing. For the CHT-Android app, the "reference" apps (`medicmobilegamma` and `unbranded`) must be published in the Play Store.
+3. [Publish]({{< ref "apps/guides/android/publishing" >}}) in the Play Store, F-Droid or whatever channel is used for publishing. For the CHT-Android app, the "reference" apps (`medicmobilegamma` and `unbranded`) must be published in the Play Store.
 4. Announce the release on the [CHT forum](https://forum.communityhealthtoolkit.org), under the "Product - Releases" category.
-
-## New flavor release
```

Comment:
Was this removed on purpose? And if yes, can you quickly explain why it's not needed anymore? (so I can understand the context 😅). Thanks

Reply:
This page was mixing two different concepts. One is for Core Developers building the next feature into the Android app, and the other is for App Developers building a flavor/brand of the app. The bit that got removed was better covered on a different existing page here: https://docs.communityhealthtoolkit.org/core/guides/android/branding/

So I removed this as duplicate.